EmployeeLeaveTracking/EmployeeLeaveTracking.Data/DTOs/OnDutyDTO.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace EmployeeLeaveTracking.Data.DTOs
+{
+    public class OnDutyDTO
```

Comment:
How this DTO is different from CompOffDTO

Reply:
Made use of only 1 DTO for this purpose